examples/BZNSYP/run.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+# Copyright 2022 Zhengxi Liu (xcmyz@outlook.com)
+
+. path.sh
+
+stage=                            # start from -1 if you need to download data
+stop_stage=
+
+dataset_dir=                      # path to dataset directory
+
+fastspeech2_outputdir=fastspeech2
+fastspeech2_config=conf/fastspeech2.yaml
+
+hifigan_outputdir=hifigan
+hifigan_config=conf/hifigan_v1.yaml
+
+conda_base=$(conda info --base)
+source $conda_base/bin/activate wetts
+
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+  # Prepare samples
+  python examples/BZNSYP/local/parse_data.py \
```

Comment:
it should be local/parse_data.py since we run the script in current directory.

Reply:
Done.